spot_driver/test/src/test_parameter_interface.cpp

```diff
@@ -143,7 +143,7 @@ TEST_F(RclcppParameterInterfaceEnvVarTest, GetSpotConfigFromEnvVars) {
 TEST_F(RclcppParameterInterfaceEnvVarTest, GetSpotConfigFromParameters) {
   // GIVEN we set all Spot config parameters to values which are different than the default values
   constexpr auto address_parameter = "192.168.100.10";
-  node_->declare_parameter("address", address_parameter);
+  node_->declare_parameter("hostname", address_parameter);
```

Comment:
do we want to change this variable too?  `address_parameter` --> `hostname_parameter`